qemu/tests/netperf_udp.py

```diff
@@ -169,9 +169,13 @@ def run(test, params, env):
                 test.error("Output format is not expected")
             throughput.append(float(line_tokens[5]))
             msg += output
-            m_size += step
     finally:
+        error_context.context(
+            "Kill and cleanup netperf on both server and client.",
+            logging.info)
         netperf_server.stop()
+        netperf_server.cleanup()
+        netperf_client.cleanup()
```

Comment:
As errors may occur during the execution of `.cleanup`, we should consider whether let them terminate the test.